api/notd/staking_manager.py

```diff
@@ -0,0 +1,68 @@
+from typing import Set
+from typing import Tuple
+
+from core import logging
+from core.queues.sqs_message_queue import SqsMessageQueue
+from core.store.retriever import StringFieldFilter
+
+from notd.messages import RefreshStakingsForCollectionMessageContent
+from notd.messages import UpdateTokenStakingMessageContent
+from notd.model import COLLECTION_CREEPZ_ADDRESS
+from notd.store.retriever import Retriever
+from notd.store.saver import Saver
+from notd.store.schema import TokenStakingsTable
+from notd.token_staking_processor import TokenStakingProcessor
+
+
+class StakingManager:
+
+    def __init__(self, retriever: Retriever,saver: Saver, tokenQueue: SqsMessageQueue, workQueue: SqsMessageQueue, tokenStakingProcessor: TokenStakingProcessor) -> None:
+        self.retriever = retriever
+        self.saver = saver
+        self.workQueue = workQueue
+        self.tokenQueue = tokenQueue
+        self.tokenStakingProcessor = tokenStakingProcessor
```

Comment:
why is this tokenStakingProcessor but the class here is StakingManager - make both token or both just staking. also the table name, api name, messaage names should all match the same thing

Reply:
I just followed the way listings and ownership where done